tutorials/dapt-curation/README.md

```diff
@@ -0,0 +1,49 @@
+# Data curation for DAPT (Domain Adaptive Pre-Training)
+
+[ChipNeMo](https://arxiv.org/pdf/2311.00176) is a chip design domain adapted LLM. LLama 2 NIM models are continually pre-trained with 20B plus tokens on domain-specific chip design data, including code, documents, etc., based on NeMo foundation models and then fine-tuned with instruction datasets from design data as well as external sources.
```

Comment:
NIMs are microservices. I don't see you using a microservice in the tutorial, so you probably are looking for a different word.

Please clarify the relationship between ChipNeMo, Llama 2, and "NeMo foundation models" in this paragraph. You introduce all of them and it's unclear what the connection is.

Reply:
Refined the text. Please review

Reply:
Thanks, remove the word "NIM" and we should be good

Reply:
Done